brand.php

```diff
@@ -100,6 +100,19 @@
         <img src="<?=$sitewide['root'];?>images/brand/community-black.png" alt="elementary Community Logo in black"/>
         <p>This helps establish the product as part of the overall elementary community while reducing confusion that can arise from using the main logomark.</p>
     </section>
+    
+    <section id="hardware">
+        <h2>Hardware Distributors</h2>
+        <p>When a user sees or experiences elementary OS, we want to be sure they are seeing it as we intended. We also want to ensure they will have a consistent experience no matter how they access it, be it by downloading it from our website or using it pre-installed on a hardware product.</p>
+        <p>A modified version of elementary OS may be re-distributed pre-installed on a hardware product so long as it meets one or more of the following conditions:</p>
+        <ol>
+            <li>It is completely rebranded, containing none of the elementary brand marks,</li>
```

Comment:
Probably should add a line stating that they need to follow the terms of the licensing of the including software, specifically the GPL